### PR TITLE
Use koji from PyPI for testing

### PR DESCRIPTION
**Pull request by chmeliik**
It seems that each environment (other than `fedora + python3`) has some issues with installing koji from PyPI, workarounds are in `test.sh`.

### CODE REVIEW

**Comment by MartinBasti** [comment]
Please add ticket ID into commit msg

**Comment by athos-ribeiro** [comment]
Nice work!
Make sure to rebase on master and verify changes are still compatible with #1191

File: test.sh [hunk 1]
```diff
@@ -102,6 +102,18 @@ if [[ $OS == "fedora" ]]; then
   $RUN $PIP install -r requirements-flatpak.txt
 fi
 
+# make sure rpm-py-installer (koji dependency) can be installed without failing
+if [[ $PYTHON_VERSION == 2* ]]; then
+  $RUN $PKG install -y gcc krb5-devel python-devel
+  # common dependencies for centos and fedora
+
+  if [[ $OS == "fedora" ]]; then
+    $RUN $PKG install -y popt-devel
```
[the remark this answers is not in the document]
Comment (chmeliik) [reply test.sh]: The full error message is:
```
Install failed without rpm-devel package by below reason.
Can you install the RPM package, and run this installer again?

Install a RPM download plugin or
install the RPM pacakge [popt-devel].
```
That is why I went with `popt-devel`, but `rpm-devel` works too (it installs `popt-devel` as a dependency). Should I use `rpm-devel` instead?

[the remark this answers is not in the document]
Comment (MartinBasti) [reply test.sh]: ok you can keep popt-devel

[the remark this answers is not in the document]
Comment (MartinBasti) [reply test.sh]: In  OMPS, we need popt-devel for py3 as well. Maybe something from py3 deps installs it, but we should explicit about dependencies

**Comment by chmeliik** [comment]
Updated PR, now installing `rpm-py-installer` dependencies on py3 as well.

**Comment by chmeliik** [comment]
After a discussion with @MartinBasti, now installing `popt-devel` on centos as well (because we couldn't figure out why it worked without it)

**Comment by MartinBasti** [comment]
> After a discussion with @MartinBasti, now installing popt-devel on centos as well (because we couldn't figure out why it worked without it)

Not just because that. koji and rpm-py-installer have clear dependencies and we shouldn't just rely on side effects that some platforms have it installed already. Explicit is better than implicit - python zen

**Comment by MartinBasti** [comment]
No Release notes needed, tests change only

**Pull request merged**